### DISCUSSION TITLE
`landmarks` role in EPUB vs `landmark` in ARIA

### DISCUSSION THREAD

**Discussion by HadrienGardeur** [readium/guided-navigation#13] [General]
In ARIA, `landmark` is an abstract superclass role for important sections of a document.

In EPUB, `landmarks` identifies a list of links where each element of the list has a unique `epub:type`, with a recommendation to always include both `bodymatter` and `toc`.

In the [EPUB Type to ARIA Role Authoring Guide](https://idpf.github.io/epub-guides/epub-aria-authoring/), the EPUB `landmarks` is mapped to `directory` which was deprecated in ARIA 1.2 in favour of `list`.

Up until now, we've sticked closely to EPUB for our usage of `landmarks`, which we already define as a collection role for RWPM: https://readium.org/webpub-manifest/roles.html#extensions

Should we continue to follow that approach or should we be wary of the fact that this term has a different meaning in the context of ARIA?

**Comment by llemeurfr** [comment]
In ARIA 1.2 I find landmarks roles [here](https://www.w3.org/TR/wai-aria-1.2/#landmark_roles). These roles are interesting in Web pages, not so much in ebooks. 

Therefore I would stick with the EPUB landmarks vocabulary, which refers to important sections of a publication.

**Reply by HadrienGardeur** [reply]
This discussion is about terminology rather than the usefulness of the ARIA landmark roles. IMO, some of these roles are worth having in our list because we may encounter them (`main`, `navigation` and `region` for example).

Terminology-wise, we're walking on a thin line between both EPUB and ARIA so far for roles, which is why we need to disambiguate when we encounter a term with different definitions in either specs.
